scala/integration/bsp/src/org/jetbrains/plugins/scala/bsp/flow/open/ScalaBspProjectOpenProcessor.scala

```diff
@@ -2,10 +2,11 @@ package org.jetbrains.plugins.scala.bsp.flow.open
 
 import org.jetbrains.plugins.bsp.flow.open.{BspProjectOpenProcessorExtension, BuildToolId}
 import org.jetbrains.plugins.scala.bsp.BspFeatureFlags
+import org.jetbrains.plugins.scala.bsp.config.ScalaPluginConstants
 
 class ScalaBspProjectOpenProcessor extends BspProjectOpenProcessorExtension {
 
   override def getShouldBspProjectOpenProcessorBeAvailable: Boolean = BspFeatureFlags.isBspPluginIntegrationEnabled
 
-  override def getBuildToolId: BuildToolId = new BuildToolId("scalabsp")
+  override def getBuildToolId: BuildToolId = new BuildToolId(ScalaPluginConstants.SYSTEM_ID.getId)
```

Comment:
Just a nit to start discussion because I'm not sure about that - did you find any resources pointing that it should be set that way? I think the JB team mentioned that it should probably be set to the "sbt" value.

Reply:
You may be right, actually. Unless they've used the same name for unrelated things, it seems to be the same as the value of the `name` field in the connection file ([[ij-bsp]](https://github.com/JetBrains/intellij-bsp/blob/3422c1ba21c5759db8738058ed3a2d67312b3206/src/main/kotlin/org/jetbrains/plugins/bsp/server/connection/BspFileConnection.kt#L111), [[ij-bazel]](https://github.com/JetBrains/intellij-bazel/blob/989f93226725cb33e7520eb55a2a0b2b09ffcc98/src/main/kotlin/org/jetbrains/bazel/flow/open/OpenBazelProjectViaBspPluginAction.kt#L57), [[bazel search]](https://github.com/search?q=repo%3AJetBrains%2Fintellij-bazel+BazelPluginConstants.ID&type=code)). I would be for changing `ScalaPluginConstants.ID` to the appropriate build tool. From what I understand, that would be `"sbt"`; however, it might also be `"bsp"` as we are to depend on ij-bsp

Reply:
I think that it shouldn't be `bsp` because `intellij-bazel` also depends on that plugin and it doesn't use `"bsp"`, I would put `"sbt"` here for now, because it will be easier to debug later on if something was wrong - but we should keep that change in mind if later on it will appear that `ID == "sbt"`. Would you be okay with that approach?

Reply:
Ok, let's leave it at that for now